android/src/main/java/com/gr4vy/embedreactnative/Gr4vyActivity.java

```diff
@@ -230,6 +250,20 @@ protected static List<CartItem> convertCartItems(String cartItemsJson) {
     return cartItemList;
   }
 
+  protected Gr4vyBuyer convertBuyer(ReadableMap source) {
+    if (source == null) {
+      return null;
+    }
+
+    Gr4vyBuyer buyer = decode(source, Gr4vyBuyer.class);
+
+    if (buyer == null) {
```

Comment:
If `decode` could return `null` on decode failure what is the point on returning null instead of just returning the buyer in line 264?

Reply:
you're right, removed